BasiliskII/src/BeOS/extfs_beos.cpp

```diff
@@ -458,3 +459,31 @@ bool extfs_rename(const char *old_path, const char *new_path)
 {
 	return rename(old_path, new_path) == 0;
 }
+
+/*
+ *  Strings (filenames) conversion
+ */
+
+// Convert from the host OS filename encoding to MacRoman
+const char *host_encoding_to_macroman(const char *filename)
+{
+	int32 state = 0;
+	static char buffer[512];
+	int32 size = 512;
```

Comment:
This still has the issue I mentioned on the other review. This should be size = sizeof(buffer) - 1;

Same for line 485.